patches/master_patch.patch

```diff
@@ -1842,6 +1842,1081 @@ index b73f65168086189aed021cc82ffef5fde1f2d061..aa3f97fa57a3cb5616cd3811a34d0698
+-    { "name": "google.sc", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.se", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.sh", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.si", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.sk", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.sm", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.sn", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.so", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.st", "include_subdomains": true, "pins": "google" },
+-    { "name": "googlesyndication.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "googletagmanager.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "googletagservices.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.td", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.tg", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.tk", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.tl", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.tm", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.tn", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.to", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.tt", "include_subdomains": true, "pins": "google" },
+-    { "name": "googleusercontent.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.us", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.uz", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.vg", "include_subdomains": true, "pins": "google" },
+-    { "name": "googlevideo.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.vu", "include_subdomains": true, "pins": "google" },
+-    { "name": "googleweblight.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "google.ws", "include_subdomains": true, "pins": "google" },
+-    { "name": "gstatic.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "static.googleadsserving.cn", "include_subdomains": true, "pins": "google" },
+-    { "name": "urchin.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "www.googlegroups.com", "include_subdomains": true, "pins": "google", "expect_ct": true, "expect_ct_report_uri": "https://clients3.google.com/ct_upload" },
+-    { "name": "youtu.be", "include_subdomains": true, "pins": "google" },
+-    { "name": "youtube-nocookie.com", "include_subdomains": true, "pins": "google" },
+-    { "name": "ytimg.com", "include_subdomains": true, "pins": "google" },
++    { "name": "2mdn.net", "include_subdomains": true },
```

Comment:
please verify if this is true, but I think all the entries without a "mode" or "pins" property would do nothing and thus can be deleted